tests are not passing

On master, `npm i && npm run test` fails with 
```
  86:5  error  componentWillUpdate is deprecated since React 16.3.0, use UNSAFE_componentWillUpdate instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillupdate  react/no-deprecated
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -36,11 +36,7 @@
   ],
   "settings": {
     "react": {
-      "createClass": "createReactClass", // Regex for Component Factory to use,
-                                         // default to "createReactClass"
-      "pragma": "React",  // Pragma to use, default to "React"
-      "version": "^15.6.1", // React version, default to the latest React stable release
-
+      "version": "^15.6.1"
     }
   }
 }

--- a/.eslintrc
+++ b/.eslintrc
@@ -33,5 +33,14 @@
   },
   "plugins": [
     "react"
-  ]
+  ],
+  "settings": {
+    "react": {
+      "createClass": "createReactClass", // Regex for Component Factory to use,
+                                         // default to "createReactClass"
+      "pragma": "React",  // Pragma to use, default to "React"
+      "version": "^15.6.1", // React version, default to the latest React stable release
+
+    }
+  }
 }

